src/core/infer_response.cc

```diff
@@ -225,6 +230,39 @@ InferenceResponse::SendWithStatus(
   return InferenceResponse::Send(std::move(response), flags);
 }
 
+void
+InferenceResponse::TraceTensor()
```

Comment:
Same comments as for InferenceRequest::TraceTensor.
Plus in both cases you need to include error handing. For now can just output something in the log will be sufficient I think,